ls1intum/Artemis
# [Exam] Added publishResultsDate, examStudentReviewStart, examStudentReviewEnd fields to Exam

### Checklist

### Motivation and Context



The goal was to include publishResultsDate, examStudentReviewStart, examStudentReviewEnd as new date fields in the exam.

### Description

- Changed Entity on Server Side
- Changed Model on Client Side
- Changed exam-update.component to include new input fields

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Create an Exam using these new fields (no validation at the moment)
4. See that no error occurs
5. Edit the exam again and see the changes to the new fields reflected in the ui

### Screenshots


![image](https://user-images.githubusercontent.com/29718932/86788661-b428ee80-c066-11ea-9f1d-53fa44de9d80.png)

![image](https://user-images.githubusercontent.com/29718932/86799315-1b986b80-c072-11ea-8680-1a8d0eaff04d.png)
